cachi2/interface/cli.py

```diff
@@ -148,6 +169,15 @@ def fetch_deps(
             "already have a vendor/ directory (will fail if changes would be made)."
         ),
     ),
+    config_file: Path = typer.Option(
+        None,
+        "--config-file",
+        help="Process the config file at this path.",
+        file_okay=True,
```

Comment:
`file_okay=True` doesn't mean it must be a file - instead, we want `dir_okay=False`

file_okay=True means "don't fail if it's a file" (rather confusing options, I know)